### ISSUE TITLE
does hybridparallelplugin  support gradient accumulation?

### ISSUE THREAD

**Issue by YixinSong-e** [hpcaitech/ColossalAI#4776]
### 🐛 Describe the bug

I want to Continue pretrain llama with only 8 A100-80G. And I want to set global batchsize to 1024. But I can't find gradient accumulation setting.

**Comment by JThh** [comment]
Kindly refer to this [guide](https://colossalai.org/docs/features/gradient_accumulation_with_booster) for enabling gradient accumulation.
